packages/pn-commons/src/utils/TimelineUtils/PrepareAnalogDomicileFailureStep.ts

```diff
@@ -0,0 +1,52 @@
+import { getLocalizedOrDefaultLabel } from '../../services/localization.service';
+import { PrepareAnalogDomicileFailureDetails } from '../../types/NotificationDetail';
+import { TimelineStep, TimelineStepInfo, TimelineStepPayload } from "./TimelineStep";
+
+export class PrepareAnalogDomicileFailureStep extends TimelineStep {
+    getTimelineStepLabel(): string {
+      return getLocalizedOrDefaultLabel(
+        'notifications',
+        `detail.timeline.prepare-analog-domicile-failure`,
+        `Aggiornamento sull'invio cartaceo`
+      );
+    }
+  
+    getTimelineStepInfo(payload: TimelineStepPayload): TimelineStepInfo | null {
+      // label - separate method just to avoid "cognitive complexity" eslint errors
+      // ///////////////////////////////////////////////////////////////
+      const label = this.getTimelineStepLabel();
+  
+      // details
+      // ///////////////////////////////////////////////////////////////
+      const failureCause = (payload.step.details as PrepareAnalogDomicileFailureDetails).failureCause || 'ZZZ';
+  
+      const addressData = this.completePhysicalAddressFromAddress((payload.step.details as PrepareAnalogDomicileFailureDetails).foundAddress);
+  
+      // eslint-disable-next-line functional/no-let
+      let description = getLocalizedOrDefaultLabel(
+        'notifications',
+        `detail.timeline.prepare-analog-domicile-failure-${failureCause}-description${
+          payload.isMultiRecipient ? '-multirecipient' : ''
+        }`,
+        '',
+        {
+          ...this.nameAndTaxId(payload),
+          ...addressData,
+        }
+      );
+  
+      if (description.length === 0) {
+        description = getLocalizedOrDefaultLabel(
+          'notifications',
+          `detail.timeline.prepare-analog-domicile-failure-XXX-description${
+            payload.isMultiRecipient ? '-multirecipient' : ''
+          }`,
+          `Non è stato trovato un indirizzo valido per predisporre un altro tentativo di invio - motivo sconosciuto.`,
```

Comment:
@clombardi i cannot find this copy here https://pagopa.atlassian.net/wiki/spaces/PN/pages/629014723/Timeline+events+e+cosa+mostrare+agli+utenti was it approved by @AliceAzzolini ?

Reply:
@ZorbaDimatteo I did, you can find in the `PREPARE_ANALOG_DOMICILE_FAILURE` row

Reply:
@AliceAzzolini I can't find this specific copy "Non è stato trovato un indirizzo valido per predisporre un altro tentativo di invio - motivo sconosciuto." I suppose is used to manage a default error